input/pagecontent/2-5-EventNotification.md

```diff
@@ -2,16 +2,16 @@ The Hub SHALL notify Subscribers of workflow-related events to which the Subscri
 
 ### Event Notification
 
-The event notification interaction include the following fields:
+The event notification message includes the following fields:
 
 {:.grid}
 Field       | Optionality | Type | Description
 ----------- | ----------- | ---- | ------------
-`timestamp` | Required    | *string* | ISO 8601-2 timestamp in UTC describing the time at which the event occurred.
+`timestamp` | Required    | *string* | ISO 8601-2 timestamp in UTC describing the time from the Request Event Notification.

```

Comment:
```suggestion
`timestamp` | Required    | *string* | ISO 8601-2 timestamp in UTC describing the time at which the event occurred.

```
I don't understand the proposed change and it feels like it's a new, different definition of this field, which we oughtn't do. Reverting.